analysis.go

```diff
@@ -2,10 +2,16 @@ package main
 
 import (
 	"encoding/json"
-	"os"
+	"io/ioutil"
 	"path/filepath"
 )
 
+var (
```

Comment:
I considered using a const, but doesn't that make code that uses this value harder to test?